@FISHDATA/personalAnalysis.m

```diff
@@ -12,6 +13,10 @@
 TrRes(:,5) = obj.ratePerformanceByTrials('crossMidline',false);
 
 % Plot results
+plotfigure(TrRes,idxPostTr);
```

Comment:
Good. We might need to use this function in the function "visualizeLearningResponses".